src/comparator/field_comparator.py

```diff
@@ -143,7 +160,10 @@ def compare(self):
                     category=FindingCategory.FIELD_TYPE_CHANGE,
                     proto_file_name=self.field_update.proto_file_name,
                     source_code_line=self.field_update.type_name.source_code_line,
-                    message=f"Type of an existing field `{self.field_original.name}` is changed from `{self.field_original.type_name.value}` to a map.",
+                    subject=self.field_original.name,
+                    context=self.context,
+                    oldtype=self.field_original.type_name.value,
```

Comment:
Is it possible to add types for key/value?

Reply:
Great idea - added, thank you!